app/Http/Controllers/CreatePostingPeriod.php

```
@@ -0,0 +1,184 @@
+use App\Models\Period;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Log;
+
+
+class CreatePostingPeriod extends Controller
+{
+    /**
+     * Handle the incoming request.
+     * Create posting period for the NEXT month
+     */
+    public function __invoke(?Request $request = null): JsonResponse
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)